resources/views/layouts/forum.blade.php

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>@yield('title','Home Page')</title>
+    <!-- Styles - Bootstrap is loaded locally -->
+    <link rel="stylesheet" type="text/css" href="{{ asset('css/customStyle.css') }}">
+    <link href="{{ asset('css/bootstrap.min.css') }}" rel="stylesheet">
+
+    <script src="{{ asset('js/jquery-3.3.1.slim.min.js') }}"></script>
+    <script src="{{ asset('js/popper.min.js') }}"></script>
+    <script src="{{ asset('js/bootstrap.min.js') }}"></script>
```

Comment:
Use cdns or install those packages via composer. Look at resources/js/app.js file. If you include that file you would have bootstrap and jquery. Reason: those are more files to maintain and to care about.

Reply:
It don't understand. Just including that file I have those functionalities? Maybe it explains why it works without those packages jaja xD

Reply:
Including app.js includes all the dependencies of app.js. You can edit this file or generate a new one with custom dependencies pretty similar to app.js

Reply:
I have some doubts about it.

What are the different between resources/js/app.js and public/js/app.js?
When I use `<script src="{{ asset('js/app.js') }}" defer></script>`, what file I am using?

PD: forget slack. This is more effective